app/src/main/java/com/infomaniak/drive/data/services/UploadWorker.kt

```diff
@@ -71,6 +71,12 @@ class UploadWorker(appContext: Context, params: WorkerParameters) : CoroutineWor
         contentResolver = applicationContext.contentResolver
         Log.d(TAG, "UploadWorker start job !")
 
+        // Checks if the maximum number of retry allowed is reached
+        if (runAttemptCount >= MAX_RETERY_COUNT) {
```

Comment:
little typo `RETRY` instead of `RETERY` ;)

Reply:
Indeed I had not seen, thank you :)